src/gui/modals/ExportProjectDialog.cpp

```diff
@@ -44,21 +45,18 @@ ExportProjectDialog::ExportProjectDialog( const QString & _file_name,
 	m_renderManager( nullptr )
 {
 	setupUi( this );
-	setWindowTitle( tr( "Export project to %1" ).arg(
-					QFileInfo( _file_name ).fileName() ) );
-
-	// Get the extension of the chosen file.
-	QStringList parts = _file_name.split( '.' );
-	QString fileExt;
-	if( parts.size() > 0 )
-	{
-		fileExt = "." + parts[parts.size()-1];
-	}
 
+	m_targetFileInfo = QFileInfo(m_fileName);
+	m_needCheckIfTargetFileExists = false;
+	QString fileExt = m_targetFileInfo.suffix();
+	m_originalTargetFileExt = "." + fileExt;
```

Comment:
These would be better set in the member initialiser list, before the constructor body.